src/webserver.cpp

```diff
@@ -42,6 +42,7 @@
 #include <QUrl>
 #include <QVector>
 #include <QDebug>
+#include <sys/socket.h>
```

Comment:
Windows does not have this header. You can remove it from here, `mongoose` already has it.